frappe/public/js/frappe/widgets/widget_dialog.js

```diff
@@ -35,7 +35,8 @@ class WidgetDialog {
 		// __("New Chart") __("New Shortcut") __("Edit Chart") __("Edit Shortcut")
 
 		let action = this.editing ? "Edit" : "Add";
-		return __(`${action} ${frappe.model.unscrub(this.type)}`);
+		let label = action = action + " " + frappe.model.unscrub(this.type);
```

Comment:
will this be translatable?

Reply:
Yes, we have added a comment above with all possible values so the parser can pick it.